helper.mac.fish

```diff
@@ -128,7 +128,7 @@ end
 function runLocal
   if test -z "$SSH_AUTH_SOCK"
     eval (ssh-agent -c) > /dev/null
-    ssh-add ~/.ssh/id_rsa
+    ssh-add ~/.ssh/id_rsa ~/.ssh/id_ed25519
```

Comment:
@goedderz would be safe to check for the existence of the key here? like in the cases above?

Reply:
Good point. Now I've glanced at the `ssh-add(1)` manpage and found

> When run without arguments, it adds the files ~/.ssh/id_rsa, ~/.ssh/id_dsa, ~/.ssh/id_ecdsa, and ~/.ssh/id_ed25519.

Perhaps we should just do that instead, and add only id_deploy manually? It's probably also future-proof, if when new authentication methods are added in the future.